spec/javascript/containers/CuisineShowContainerSpec.js

```diff
@@ -41,25 +44,56 @@ describe('CuisineShowContainer', () => {
       }, 0);
     });
 
-    it('renders a list item for each item returned from the api call', (done) => {
+    it('renders a address list item for each item returned from the api call', (done) => {
       setTimeout(() => {
         expect(wrapper.find('li.rest-address').text()).toBe('52 Shady Lane');
         done();
       }, 0);
     });
```

Comment:
Actually, the CuisineShowContainer does not render these items, the `RestaurantListTile` does. You instead want to test that CuisineShowContainer renders RestaurantListTiles, per the enzyme reading on containers

Reply:
will do

Reply:
Adjusted the Enzyme testing to the best of my abilities and per my understanding of your advice.